manifests/a/Atticus64/juice/0.4.0/Atticus64.juice.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Atticus64.juice
+PackageVersion: 0.4.0
+InstallerLocale: en-US
+InstallerType: portable
+InstallModes:
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+Commands:
+- juice
+Protocols:
+- http
+FileExtensions:
+- json
+ReleaseDate: 2023-02-20
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Atticus64/juice/releases/download/v0.4.0/juice.exe
+  InstallerSha256: A120531EFA64DE0E5FF0EFF2B5CE24DD3DD548EFAF03182EE57EC536D2533DEC
```

Hi @Atticus64, this is the correct hash for the installer.

```suggestion
  InstallerSha256: 9808B307F56701791F5838943D772C77A6B234F4FC8247074AD9F1C1F19C09CC
```